src/controllers/EmailVerification.js

```diff
@@ -0,0 +1,30 @@
+/**=====================================================================
+ * ====   TITLE:: IMPLEMENTATION OF EMAIL VERIFICATION CONTROLLER   ====  
+ * ====   AUTHOR:: HACKINWALE <hackinwale.developer@gmail.com>      ====
+ * ====   DATE::            21TH JUNE 2020                          ====
+ * =====================================================================
```

Comment:
Awesome comment style, but i'm not sure about the convention though, perhaps we can use this
https://google.github.io/styleguide/jsguide.html#jsdoc-method-and-function-comments

Reply:
This is just my custom comment style.

This comment is not applicable to functions or class or any other parameter in the code. Just my style and it affects nothing